notebooks/access_github_ci_data.ipynb

```diff
@@ -68,17 +68,30 @@
     "TOKEN = os.getenv(\"GITHUB_ACCESS_TOKEN\")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "842fbe80-3d54-4dfb-83c3-c18c5cdcda89",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# mode can be \"check or \"workflow\" depending on whether you want to collect checks data or workflows data\n",
+    "MODE = \"check\""
```

Comment:
Does it make sense to call this "commits" instead of "checks"?

Reply:
Was calling the modes "checks" & "workflows" as the 2 APIs I was using to collect are the  https://docs.github.com/en/rest/checks/runs or https://docs.github.com/en/rest/actions/workflows 
I did not see an easy way to get the checks data without extracting the commit refs first, so hence getting all the commits first